CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#921](https://github.com/nix-rust/nix/pull/921))
 - Added support for `SCM_CREDENTIALS`, allowing to send process credentials over Unix sockets.
   ([#923](https://github.com/nix-rust/nix/pull/923))
+- Added a `statfs` wrapper.
```

Comment:
You didn't add a `statfs` wrapper because it already existed.  You should remove this changelog entry, but leave the entry in the Changed section.